plugins/proxy-backend/src/service/router.test.ts

```diff
@@ -35,3 +52,151 @@ describe('createRouter', () => {
     expect(router).toBeDefined();
   });
 });
+
+describe('buildMiddleware', () => {
+  const logger = winston.createLogger();
```

Comment:
Does this logger no longer give warnings? I seem to remember that a logger with no sinks outputs log noise during tests to warn about possible misconfiguration.

There's getVoidLogger() in backend-common if you want for this purpose, maybe it's not necessary anymore

Reply:
I didn't see any warnings and I just copied it from other tests.